index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Designer and Prospective Developer in Midland, TX</title>
+  <meta property="og:description" content="My name is Joe Aylor and I live for the
+  creative solution for your idea. Constantly learning and ideating about the next great thing! ">
+  <meta property="og:locale" content="en_US" />
+  <meta name="keywords" content="design,graphic-design,website,developer,advertising,texas" />
+  <meta name="og:url" content="http://www.joeaylor.com" />
+  <meta name="og:site_name" content="Joe Aylor" />
+  <!-- links to stuff -->
+  <link rel="stylesheet" href="assets/css/style.css" />
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
+  <!-- Fonts -->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+</head>
+
+<header>
+
+  <!-- Navbar -->
+<div id="Menu">
+    <div class="logo">
+        <h1 class="top-logo"><a href="index.html">Joe Aylor</a></h1>
+      <ul>
+        <li>Menu 1</li>
+        <li>Menu 2</li>
+        <li>Menu 3</li>
+        <li>Menu 4</li>
+      </ul>
+    </div>
+  </div>
+</header>
+
+<body>
+  <!-- single image box -->
+    <div class="first-box image-box ">
+      <div class="color-overlay">
+        <div class="text-box-left white-text">
+          <h1>Image 1</h1>
+          <p>This text is over a full-width image and should take up half the width of the page. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum.</p>
+          <p class="link-button">
```

Comment:
I know you mentioned you haven't done any linking yet, but just want to clarify that any button / link type stiles should usually only go on an `<a>` tag.

The reason being accessibility. If someone that's visually impaired is using your site, their software knows that `<p>` tags are ONLY meant to display text - and `<a>` tags are meant to be clicked. Their software would think there aren't any links on your site.

So this code would be changed to:

```html
<p><a href="#" class="link-button">
  More Info
</a></p>
```